### ISSUE TITLE
"module 'pyparsing' has no attribute 'Opt'", extension is not working with SD.Next

### ISSUE THREAD

**Issue by Manchovies** [adieyal/sd-dynamic-prompts#718]
Was working just a few days ago.. not sure what happened. I will be here to try any troubleshooting steps or anything you want/need me to. Thanks in advance and in retrospect as well for this awesome extension. 
[SD -DYNAMIC-PROMPTS ERROR.txt](https://github.com/adieyal/sd-dynamic-prompts/files/14028165/SD.-DYNAMIC-PROMPTS.ERROR.txt)

Here is my log (it is also attached as a .txt file): 

```
Using VENV: C:\SDNext\automatic\venv
12:42:13-813796 INFO     Starting SD.Next
12:42:13-818801 INFO     Logger: file="C:\SDNext\automatic\sdnext.log" level=DEBUG size=65 mode=create
12:42:13-822803 INFO     Python 3.10.6 on Windows
12:42:13-979364 INFO     Version: app=sd.next updated=2024-01-22 hash=f0ec1a1c
                         url=https://github.com/vladmandic/automatic.git/tree/dev
12:42:14-287193 INFO     Latest published version: 5fb290f443d5f38a5c9f6e6095aabeab8e3a991d 2024-01-13T13:47:29Z
12:42:14-295705 INFO     Platform: arch=AMD64 cpu=AMD64 Family 23 Model 113 Stepping 0, AuthenticAMD system=Windows
                         release=Windows-10-10.0.22631-SP0 python=3.10.6
12:42:14-300551 DEBUG    Setting environment tuning
12:42:14-301565 DEBUG    HF cache folder: C:\Users\sonic\.cache\huggingface\hub
12:42:14-302562 DEBUG    Torch overrides: cuda=False rocm=False ipex=False diml=False openvino=False
12:42:14-305567 DEBUG    Torch allowed: cuda=True rocm=True ipex=True diml=True openvino=True
12:42:14-311564 INFO     nVidia CUDA toolkit detected: nvidia-smi present
12:42:17-741077 DEBUG    Repository update time: Mon Jan 22 14:18:40 2024
12:42:17-744076 INFO     Startup: standard
12:42:17-745076 INFO     Verifying requirements
12:42:17-802094 INFO     Verifying packages
12:42:17-805093 INFO     Verifying submodules
12:42:20-227021 DEBUG    Submodule: extensions-builtin/sd-extension-chainner / main
12:42:20-320766 DEBUG    Submodule: extensions-builtin/sd-extension-system-info / main
12:42:20-402670 DEBUG    Submodule: extensions-builtin/sd-webui-agent-scheduler / main
12:42:20-495749 DEBUG    Submodule: extensions-builtin/sd-webui-controlnet / main
12:42:20-605497 DEBUG    Submodule: extensions-builtin/stable-diffusion-webui-images-browser / main
12:42:20-692225 DEBUG    Submodule: extensions-builtin/stable-diffusion-webui-rembg / master
12:42:20-777061 DEBUG    Submodule: modules/k-diffusion / master
12:42:20-859905 DEBUG    Submodule: modules/lora / main
12:42:20-952021 DEBUG    Submodule: wiki / master
12:42:21-006669 DEBUG    Register paths
12:42:21-197116 DEBUG    Installed packages: 269
12:42:21-200115 DEBUG    Extensions all: ['Lora', 'sd-extension-chainner', 'sd-extension-system-info',
                         'sd-webui-agent-scheduler', 'sd-webui-controlnet', 'stable-diffusion-webui-images-browser',
                         'stable-diffusion-webui-rembg']
12:42:21-573954 DEBUG    Running extension installer:
                         C:\SDNext\automatic\extensions-builtin\sd-extension-system-info\install.py
12:42:22-238091 DEBUG    Running extension installer:
                         C:\SDNext\automatic\extensions-builtin\sd-webui-agent-scheduler\install.py
12:42:22-904991 DEBUG    Running extension installer:
                         C:\SDNext\automatic\extensions-builtin\sd-webui-controlnet\install.py
12:42:23-688720 DEBUG    Running extension installer:
                         C:\SDNext\automatic\extensions-builtin\stable-diffusion-webui-images-browser\install.py
12:42:24-337482 DEBUG    Running extension installer:
                         C:\SDNext\automatic\extensions-builtin\stable-diffusion-webui-rembg\install.py
12:42:24-988827 DEBUG    Extensions all: ['sd-dynamic-prompts', 'sd-history-slider', 'sd-webui-prompt-all-in-one',
                         'sd_webui_SAG', 'stable-diffusion-webui-wildcards', 'test_my_prompt']
12:42:24-994843 DEBUG    Running extension installer: C:\SDNext\automatic\extensions\sd-dynamic-prompts\install.py
12:42:25-890687 DEBUG    Running extension installer:
                         C:\SDNext\automatic\extensions\sd-webui-prompt-all-in-one\install.py
12:42:34-129298 INFO     Extensions enabled: ['Lora', 'sd-extension-chainner', 'sd-extension-system-info',
                         'sd-webui-agent-scheduler', 'sd-webui-controlnet', 'stable-diffusion-webui-images-browser',
                         'stable-diffusion-webui-rembg', 'sd-dynamic-prompts', 'sd-history-slider',
                         'sd-webui-prompt-all-in-one', 'sd_webui_SAG', 'stable-diffusion-webui-wildcards',
                         'test_my_prompt']
12:42:34-135810 INFO     Verifying requirements
12:42:34-150331 DEBUG    Setup complete without errors: 1706031754
12:42:34-166595 INFO     Extension preload: {'extensions-builtin': 0.0, 'extensions': 0.01}
12:42:34-169591 DEBUG    Starting module: <module 'webui' from 'C:\\SDNext\\automatic\\webui.py'>
12:42:34-172590 INFO     Command line args: ['--debug'] debug=True
12:42:34-174591 DEBUG    Env flags: []
12:42:38-927009 INFO     Load packages: torch=2.1.2+cu121 diffusers=0.25.1 gradio=3.43.2
12:42:40-250215 DEBUG    Read: file="config.json" json=32 bytes=1523 time=0.000
12:42:40-254214 INFO     Engine: backend=Backend.DIFFUSERS compute=cuda device=cuda attention="xFormers" mode=no_grad
12:42:40-320287 INFO     Device: device=NVIDIA GeForce RTX 2070 SUPER n=1 arch=sm_90 cap=(7, 5) cuda=12.1 cudnn=8801
                         driver=546.33
12:42:41-532019 DEBUG    Importing LDM
12:42:41-564381 DEBUG    Entering start sequence
12:42:41-614919 DEBUG    Initializing
12:42:41-744358 INFO     Available VAEs: path="D:\StableDiffusionModelsandSymlinkStuff\models\VAE" items=2
12:42:41-747361 INFO     Disabled extensions: ['sd-webui-controlnet']
12:42:41-940940 DEBUG    Scanning diffusers cache: D:\StableDiffusionModelsandSymlinkStuff\models\Diffusers
                         D:\StableDiffusionModelsandSymlinkStuff\models\Diffusers items=4 time=0.18
12:42:41-944939 DEBUG    Read: file="cache.json" json=1 bytes=568 time=0.000
12:42:41-948962 DEBUG    Read: file="metadata.json" json=41 bytes=34077 time=0.001
12:42:42-149757 INFO     Available models: path="D:\StableDiffusionModelsandSymlinkStuff\models\Stable-diffusion"
                         items=38 time=0.39
12:42:42-274746 DEBUG    Load extensions
12:42:42-567493 INFO     Extension: script='extensions-builtin\Lora\scripts\lora_script.py'
                         12:42:42-563493 INFO     LoRA networks: available=7
                         folders=2
12:42:43-136770 INFO     Extension: script='extensions-builtin\sd-webui-agent-scheduler\scripts\task_scheduler.py'
                         Using sqlite file: extensions-builtin\sd-webui-agent-scheduler\task_scheduler.sqlite3
12:42:43-494210 ERROR    Module load: extensions\sd-dynamic-prompts\scripts\dynamic_prompting.py: AttributeError
┌───────────────────────────────────────── Traceback (most recent call last) ─────────────────────────────────────────┐
│ C:\SDNext\automatic\modules\script_loading.py:29 in load_module                                                     │
│                                                                                                                     │
│   28 │   │   │   │   with contextlib.redirect_stdout(io.StringIO()) as stdout:                                      │
│ > 29 │   │   │   │   │   module_spec.loader.exec_module(module)                                                     │
│   30 │   │   │   setup_logging() # reset since scripts can hijaack logging                                          │
│ in exec_module:883                                                                                                  │
│ in _call_with_frames_removed:241                                                                                    │
│                                                                                                                     │
│ C:\SDNext\automatic\extensions\sd-dynamic-prompts\scripts\dynamic_prompting.py:3 in <module>                        │
│                                                                                                                     │
│   2                                                                                                                 │
│ > 3 from sd_dynamic_prompts.dynamic_prompting import Script                                                         │
│   4                                                                                                                 │
│                                                                                                                     │
│ C:\SDNext\automatic\extensions\sd-dynamic-prompts\sd_dynamic_prompts\dynamic_prompting.py:13 in <module>            │
│                                                                                                                     │
│    12 from dynamicprompts.generators.promptgenerator import GeneratorException                                      │
│ >  13 from dynamicprompts.parser.parse import ParserConfig                                                          │
│    14 from dynamicprompts.wildcards import WildcardManager                                                          │
│                                                                                                                     │
│ C:\SDNext\automatic\venv\lib\site-packages\dynamicprompts\parser\parse.py:65 in <module>                            │
│                                                                                                                     │
│    64                                                                                                               │
│ >  65 OPT_WS = pp.Opt(pp.White())  # Optional whitespace                                                            │
│    66                                                                                                               │
└─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┘
AttributeError: module 'pyparsing' has no attribute 'Opt'
12:42:43-852583 INFO     Extension: script='extensions\sd-webui-prompt-all-in-one\scripts\on_app_started.py'
                         sd-webui-prompt-all-in-one background API service started successfully.
12:42:43-873087 ERROR    Module load: extensions\test_my_prompt\scripts\test_my_prompt_custom_script.py: ImportError
┌───────────────────────────────────────── Traceback (most recent call last) ─────────────────────────────────────────┐
│ C:\SDNext\automatic\modules\script_loading.py:29 in load_module                                                     │
│                                                                                                                     │
│   28 │   │   │   │   with contextlib.redirect_stdout(io.StringIO()) as stdout:                                      │
│ > 29 │   │   │   │   │   module_spec.loader.exec_module(module)                                                     │
│   30 │   │   │   setup_logging() # reset since scripts can hijaack logging                                          │
│ in exec_module:883                                                                                                  │
│ in _call_with_frames_removed:241                                                                                    │
│                                                                                                                     │
│ C:\SDNext\automatic\extensions\test_my_prompt\scripts\test_my_prompt_custom_script.py:4 in <module>                 │
│                                                                                                                     │
│    3 from PIL import Image, ImageFont, ImageDraw, ImageOps                                                          │
│ >  4 from modules.paths_internal import roboto_ttf_file                                                             │
│    5 import modules.scripts as scripts                                                                              │
└─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┘
ImportError: cannot import name 'roboto_ttf_file' from 'modules.paths_internal' (C:\SDNext\automatic\modules\paths_internal.py)
12:42:43-895487 INFO     Extensions init time: 1.62 example.py=0.05 img2imgalt.py=0.12 sd-extension-chainner=0.06
                         sd-webui-agent-scheduler=0.50 stable-diffusion-webui-images-browser=0.31
                         sd-dynamic-prompts=0.12 sd-webui-prompt-all-in-one=0.29
12:42:43-922820 DEBUG    Read: file="html/upscalers.json" json=4 bytes=2672 time=0.000
12:42:43-957847 DEBUG    Read: file="extensions-builtin\sd-extension-chainner\models.json" json=24 bytes=2719
                         time=0.000
12:42:43-964853 DEBUG    chaiNNer models: path="D:\StableDiffusionModelsandSymlinkStuff\models\chaiNNer" defined=24
                         discovered=0 downloaded=0
12:42:43-970845 DEBUG    Upscaler type=ESRGAN folder="D:\StableDiffusionModelsandSymlinkStuff\models\ESRGAN"
                         model="4x-UltraMix_Balanced"
                         path="D:\StableDiffusionModelsandSymlinkStuff\models\ESRGAN\4x-UltraMix_Balanced.pth"
12:42:43-974849 DEBUG    Upscaler type=ESRGAN folder="D:\StableDiffusionModelsandSymlinkStuff\models\ESRGAN"
                         model="4x_Nickelback_70000G"
                         path="D:\StableDiffusionModelsandSymlinkStuff\models\ESRGAN\4x_Nickelback_70000G.pth"
12:42:43-980847 DEBUG    Upscaler type=ESRGAN folder="D:\StableDiffusionModelsandSymlinkStuff\models\ESRGAN"
                         model="4x_NMKD-Siax_200k"
                         path="D:\StableDiffusionModelsandSymlinkStuff\models\ESRGAN\4x_NMKD-Siax_200k.pth"
12:42:43-988361 DEBUG    Load upscalers: total=55 downloaded=4 user=3 time=0.08 ['None', 'Lanczos', 'Nearest',
                         'ChaiNNer', 'ESRGAN', 'LDSR', 'RealESRGAN', 'SCUNet', 'SD', 'SwinIR']
12:42:44-014730 DEBUG    Load styles: folder="models\styles" items=288 time=0.02
12:42:44-023790 DEBUG    Creating UI
12:42:44-025730 INFO     Load UI theme: name="amethyst-nightfall" style=Auto base=sdnext.css
12:42:44-046246 DEBUG    UI initialize: txt2img
12:42:44-074759 DEBUG    Read: file="html\reference.json" json=32 bytes=17153 time=0.001
12:42:44-265814 DEBUG    Extra networks: page='model' items=70 subfolders=20 tab=txt2img
                         folders=['D:\\StableDiffusionModelsandSymlinkStuff\\models\\Stable-diffusion',
                         'D:\\StableDiffusionModelsandSymlinkStuff\\models\\Diffusers', 'models\\Reference'] list=0.04
                         thumb=0.01 desc=0.01 info=0.00 workers=4
12:42:44-289322 DEBUG    Extra networks: page='style' items=288 subfolders=1 tab=txt2img folders=['models\\styles',
                         'html'] list=0.03 thumb=0.00 desc=0.00 info=0.00 workers=4
12:42:44-296325 DEBUG    Extra networks: page='embedding' items=0 subfolders=0 tab=txt2img
                         folders=['D:\\StableDiffusionModelsandSymlinkStuff\\models\\embeddings'] list=0.00 thumb=0.00
                         desc=0.00 info=0.00 workers=4
12:42:44-303325 DEBUG    Extra networks: page='hypernetwork' items=0 subfolders=0 tab=txt2img
                         folders=['D:\\StableDiffusionModelsandSymlinkStuff\\models\\hypernetworks'] list=0.00
                         thumb=0.00 desc=0.00 info=0.00 workers=4
12:42:44-311325 DEBUG    Extra networks: page='vae' items=2 subfolders=0 tab=txt2img
                         folders=['D:\\StableDiffusionModelsandSymlinkStuff\\models\\VAE'] list=0.01 thumb=0.01
                         desc=0.00 info=0.00 workers=4
12:42:44-318328 DEBUG    Extra networks: page='lora' items=7 subfolders=0 tab=txt2img
                         folders=['D:\\StableDiffusionModelsandSymlinkStuff\\models\\Lora',
                         'D:\\StableDiffusionModelsandSymlinkStuff\\models\\LyCORIS'] list=0.01 thumb=0.00 desc=0.00
                         info=0.00 workers=4
12:42:44-390366 DEBUG    UI initialize: img2img
12:42:44-629701 DEBUG    UI initialize: control models=D:\StableDiffusionModelsandSymlinkStuff\models\control
12:42:44-928705 DEBUG    Read: file="ui-config.json" json=0 bytes=2 time=0.001
12:42:45-132965 DEBUG    Themes: builtin=11 default=5 external=55
12:42:47-155715 DEBUG    Extension list: processed=329 installed=13 enabled=12 disabled=1 visible=329 hidden=0
12:42:47-604742 INFO     Local URL: http://127.0.0.1:7860/
12:42:47-606743 DEBUG    Gradio functions: registered=2066
12:42:47-607743 INFO     Initializing middleware
12:42:47-614748 DEBUG    Creating API
12:42:47-992155 INFO     [AgentScheduler] Task queue is empty
12:42:47-996156 INFO     [AgentScheduler] Registering APIs
12:42:48-137466 DEBUG    Scripts setup: ['X/Y/Z Grid:0.011', 'AnimateDiff:0.008']
12:42:48-140468 DEBUG    Model metadata: file="metadata.json" no changes
12:42:48-143466 DEBUG    Model requested: fn=<lambda>
12:42:48-146472 INFO     Select: model="juggernautXL_v8Rundiffusion [aeb7e9e689]"
12:42:48-148472 DEBUG    Load model: existing=False
                         target=D:\StableDiffusionModelsandSymlinkStuff\models\Stable-diffusion\juggernautXL_v8Rundiffu
                         sion.safetensors info=None
12:42:48-232199 DEBUG    Desired Torch parameters: dtype=FP16 no-half=False no-half-vae=False upscast=False
12:42:48-235201 INFO     Setting Torch parameters: device=cuda dtype=torch.float16 vae=torch.float16 unet=torch.float16
                         context=no_grad fp16=True bf16=None
12:42:48-239200 INFO     Loading VAE:
                         model=D:\StableDiffusionModelsandSymlinkStuff\models\VAE\sdxl_vae_FP16.safetensors
                         source=settings
12:42:48-242201 DEBUG    Diffusers VAE load config: {'low_cpu_mem_usage': False, 'torch_dtype': torch.float16,
                         'use_safetensors': True, 'variant': 'fp16'}
12:42:48-246206 INFO     Autodetect: vae="Stable Diffusion XL" class=StableDiffusionXLPipeline
                         file="D:\StableDiffusionModelsandSymlinkStuff\models\Stable-diffusion\juggernautXL_v8Rundiffus
                         ion.safetensors" size=6776MB
12:42:53-484878 DEBUG    Diffusers loading:
                         path="D:\StableDiffusionModelsandSymlinkStuff\models\Stable-diffusion\juggernautXL_v8Rundiffus
                         ion.safetensors"
12:42:53-488880 INFO     Autodetect: model="Stable Diffusion XL" class=StableDiffusionXLPipeline
                         file="D:\StableDiffusionModelsandSymlinkStuff\models\Stable-diffusion\juggernautXL_v8Rundiffus
                         ion.safetensors" size=6776MB
12:44:23-066628 INFO     MOTD: N/A
12:44:36-229565 DEBUG    Themes: builtin=11 default=5 external=55
12:44:39-869496 INFO     Browser session: user=None client=69.174.95.218 agent=Mozilla/5.0 (Windows NT 10.0; Win64;
                         x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/120.0.0.0 Safari/537.36
12:45:05-747376 DEBUG    Setting model: pipeline=StableDiffusionXLPipeline config={'low_cpu_mem_usage': True,
                         'torch_dtype': torch.float16, 'load_connected_pipeline': True, 'variant': 'fp16',
                         'extract_ema': True, 'force_zeros_for_empty_prompt ': False, 'requires_aesthetics_score':
                         False, 'use_safetensors': True}
12:45:05-759377 DEBUG    Setting model VAE: name=sdxl_vae_FP16.safetensors
12:45:05-761376 DEBUG    Setting model: enable VAE slicing
12:45:05-766374 DEBUG    Setting model: enable VAE tiling
12:45:06-177885 DEBUG    Read: file="html\reference.json" json=32 bytes=17153 time=0.006
INFO:nncf:NNCF initialized successfully. Supported frameworks detected: torch, tensorflow, onnx
WARNING:nncf:NNCF provides best results with torch==2.1.0, while current torch version is 2.1.2+cu121. If you encounter issues, consider switching to torch==2.1.0
12:45:30-768170 INFO     Compress Weights: time=21.36
12:45:30-852488 INFO     Load embeddings: loaded=0 skipped=0 time=0.04
12:45:30-857490 INFO     High memory utilization: GPU=100% RAM=10% {'ram': {'used': 1.6, 'total': 15.92}, 'gpu':
                         {'used': 8.0, 'total': 8.0}, 'retries': 0, 'oom': 0}
12:45:31-185930 DEBUG    GC: collected=138 device=cuda {'ram': {'used': 1.31, 'total': 15.92}, 'gpu': {'used': 5.51,
                         'total': 8.0}, 'retries': 0, 'oom': 0} time=0.33
12:45:31-212928 INFO     Load model: time=162.70 load=162.70 native=512 {'ram': {'used': 1.32, 'total': 15.92}, 'gpu':
                         {'used': 5.51, 'total': 8.0}, 'retries': 0, 'oom': 0}
12:45:31-222444 DEBUG    Save: file="config.json" json=32 bytes=1469 time=0.001
12:45:31-227065 DEBUG    Script callback init time: image_browser.py:ui_tabs=0.39 system-info.py:app_started=0.07
                         task_scheduler.py:app_started=0.15
12:45:31-232069 INFO     Startup time: 177.04 torch=3.14 gradio=1.54 diffusers=0.07 libraries=2.60 samplers=0.10
                         vae=0.08 extensions=1.62 models=0.40 face-restore=0.12 upscalers=0.10 ui-en=0.62
                         ui-control=0.07 ui-settings=0.47 ui-extensions=1.80 ui-defaults=0.08 launch=0.35 api=0.30
                         app-started=0.23 checkpoint=163.08
12:46:00-423514 DEBUG    Server: alive=True jobs=1 requests=92 uptime=200 memory=1.0/15.92 backend=Backend.DIFFUSERS
                         state=idle
12:48:00-195282 DEBUG    Server: alive=True jobs=1 requests=94 uptime=320 memory=1.0/15.92 backend=Backend.DIFFUSERS
                         state=idle
12:50:00-107248 DEBUG    Server: alive=True jobs=1 requests=96 uptime=440 memory=1.0/15.92 backend=Backend.DIFFUSERS
                         state=idle
12:52:00-017024 DEBUG    Server: alive=True jobs=1 requests=98 uptime=560 memory=0.98/15.92 backend=Backend.DIFFUSERS
                         state=idle
12:53:59-873316 DEBUG    Server: alive=True jobs=1 requests=100 uptime=680 memory=0.98/15.92 backend=Backend.DIFFUSERS
                         state=idle
12:54:36-658892 INFO     MOTD: N/A
12:54:47-073176 DEBUG    Themes: builtin=11 default=5 external=55
12:54:52-070690 INFO     Browser session: user=None client=69.174.95.218 agent=Mozilla/5.0 (Windows NT 10.0; Win64;
                         x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/120.0.0.0 Safari/537.36
12:55:03-796677 DEBUG    Paste prompt: type="params" prompt="a deep extremely dark black vantablack (metal {rose|calle
                         lily|Chrysanthemum|begonia|crocus|peony|pansy|orchid}:1.3) with only a few
                         {blue|purple|gold|scarlet red|deep mahogany|pink} highlights on the petals on a lighter dark
                         striped background with parallel and diagonal stripes, cinematic lighting, minimalist art,
                         professional art photography, (metallic sculpture:1.1), (skewed perspective:1.2)
                         Steps: 20, Seed: 3374252586, Sampler: Default, CFG scale: 10, Size: 768x1344, Parser: Full
                         parser, Model: C-Drive-juggernautXL_v8Rundiffusion, Model hash: aeb7e9e689, VAE:
                         sdxl_vae_FP16, Prompt2: "extremely dark black flower with slight colorful highlights, dark
                         background with diagonal strips", Negative2: "deviantart, ugly", Backend: Diffusers, App:
                         SD.Next, Version: f0ec1a1, Operations: txt2img, Pipeline: StableDiffusionXLPipeline"
12:55:03-809432 DEBUG    Settings overrides: []
12:55:08-985761 DEBUG    Paste prompt: type="current" prompt="a deep extremely dark black vantablack (metal {rose|calle
                         lily|Chrysanthemum|begonia|crocus|peony|pansy|orchid}:1.3) with only a few
                         {blue|purple|gold|scarlet red|deep mahogany|pink} highlights on the petals on a lighter dark
                         striped background with parallel and diagonal stripes, cinematic lighting, minimalist art,
                         professional art photography, (metallic sculpture:1.1), (skewed perspective:1.2)"
12:55:08-993176 DEBUG    Settings overrides: []
12:55:12-689279 DEBUG    Read: file="html\reference.json" json=32 bytes=17153 time=0.000
12:55:20-401934 DEBUG    Diffuser pipeline: StableDiffusionXLPipeline task=DiffusersTaskType.TEXT_2_IMAGE
                         set={'prompt_embeds': torch.Size([1, 154, 2048]), 'pooled_prompt_embeds': torch.Size([1,
                         1280]), 'negative_prompt_embeds': torch.Size([1, 154, 2048]), 'negative_pooled_prompt_embeds':
                         torch.Size([1, 1280]), 'guidance_scale': 10, 'generator': device(type='cuda'), 'output_type':
                         'latent', 'num_inference_steps': 20, 'eta': 1.0, 'guidance_rescale': 0.7, 'denoising_end':
                         None, 'width': 768, 'height': 1344, 'parser': 'Full parser'}
Progress  4.19s/it ███▍                                10% 2/20 00:09 01:15 Base12:55:30-477152 DEBUG    Load VAE decode approximate: model="C:\SDNext\automatic\models\VAE-approx\model.pt"
Progress  1.17s/it █████████████████████████████████ 100% 20/20 00:23 00:00 Base
12:55:56-116389 DEBUG    Saving: image="outputs\text\00068-juggernautXL_v8Rundiffusion-a deep extremely dark black
                         vantablack metal rose.jpg" type=JPEG resolution=768x1344 size=0
12:55:56-130985 INFO     Processed: images=1 time=36.99 its=0.54 memory={'ram': {'used': 1.84, 'total': 15.92}, 'gpu':
                         {'used': 5.46, 'total': 8.0}, 'retries': 0, 'oom': 0}
12:55:59-787082 DEBUG    Server: alive=True jobs=1 requests=235 uptime=800 memory=1.82/15.92 backend=Backend.DIFFUSERS
                         state=idle
12:57:59-613930 DEBUG    Server: alive=True jobs=1 requests=243 uptime=919 memory=1.54/15.92 backend=Backend.DIFFUSERS
                         state=idle
```

**Comment by LankyPoet** [comment]
I am receiving the same error regarding 
"module 'pyparsing' has no attribute 'Opt'" with SD.Next.

This is one of my absolute favorite extensions, I am hopeful this can be addressed!

**Comment by LankyPoet** [comment]
PS one dev looked and suggested this? Not sure if it will help:
`dynamic-prompts doesn't have self-contained code, but instead installs and uses its own library called `dynamicprompts`. and that library uses 3rd party library called `pyparsing` - and in that entire chain of dependencies, it never checks versions. so i'm guessing `sd-dynamic-prompts` vs `dyanamicprompts` vs `pyparsing` need to match compatible versions`

**Comment by akx** [comment]
That error means there's an incompatible version of `pyparsing` installed in your environment. 

You can try to fix that by installing `pyparsing==3.0.9` (e.g. `pip install pyparsing==3.0.9`), but that may break something else.
